styles.css

```diff
@@ -33,7 +33,44 @@ img{
 }
 
 @media(max-width:600px){
+  header{
+    order: 1;
+  }
   .main{
-    background:red;
+    order: 2;
+  }
+  aside{
+    order: 3;
+  }
+  footer{
+    order: 4;
   }
 }
+
+header h1{
+  margin: 0;
+}
+
+footer h3{
+  margin: 0;
+}
+
+footer p{
+  margin: 0;
+}
+
+aside h2{
+  margin: 0;
+}
+
+main h2{
+  margin: 0;
+}
+
+main img{
+  margin: 0;
+}
+
+main div{
+  margin: 0;
+}
```

Comment:
This technically works, but a much better way of writing it is to take advantage of CSS's direct child syntax.

```css
header > * {
  /* style goes here */
}
```
> `>` means you are targeting the direct child of something. `*` means you are targeting everything. When combined in the above example, you targeting all of `header`'s direct children.